lib/plugins/aws/remove/lib/stack.js

```diff
@@ -26,8 +27,47 @@ module.exports = {
       .then(() => cfData);
   },
 
+  disassociateUsagePlan() {
```

Comment:
> @pmuens I created an example CF stack and added a question about removal to the AWS Developer Forum. It seems that it is not normal behavior that Usage Plan prevents stack deletion and they will look into it. I don't know how soon they can provide a solution, though.

Great @laardee 👍 !

Thanks for researching! Glad to hear that it's an unusual behavior since this could make the implementation way easier (once the problem is resolved).